packages/plugin-toolkit/src/cli/rollupConfig.cts

```diff
@@ -0,0 +1,39 @@
+import nodeResolve from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+import { OutputOptions, Plugin, RollupOptions } from "rollup";
+import { terser } from "rollup-plugin-terser";
+
+export const DIST_CONFIG: [RollupOptions, OutputOptions] = [
+  {
+    input: "./src/plugin.ts",
+    plugins: [nodeResolve() as Plugin, typescript() as Plugin],
+    onwarn(warning, defaultHandler) {
+      const ignoreCodes = ["TS7006", "TS7031"];
```

Comment:
So I've struggled with this for an hour or two but `tsc` does not emit error while `rollup` does. The actual bundle looks fine so for now I gave up and suppressed the warning. Since it's now a package we can update this later and publish a proper fix when it becomes important.